packages/api-server/src/database/portals.ts

```diff
@@ -0,0 +1,5 @@
+import { db } from '../db'
+
+export const getServerPortals = async (serverId: number) =>
+  (await db.dbQuery('SELECT * FROM portals WHERE server_id = $1;', [serverId]))
+    .rows
```

Comment:
This could probably go into `shared/data-models` as well to make it a little easier to keep everything together and build a little self made ORM (I hate real ORMs). 

But I've been putting all the portal stuff directly in the API.